suite-native/module-accounts-management/src/components/CoinPriceCard.tsx

```diff
@@ -14,7 +14,7 @@ type CoinPriceCardProps = {
 };
 
 type PriceChangeIndicatorProps = {
-    valuePercentageChange: number;
+    valuePercentageChange: number | null;
```

Comment:
I believe it doesn't make sense to render PriceChangeIndicator if valuePercentageChange is null

Reply:
That is one of requirements, to show skeleton (inside PriceChangeIndicator to also include title label) in case of loading.